x-pack/legacy/plugins/cross_cluster_replication/public/extend_index_management/index.js

```diff
@@ -19,4 +19,4 @@ export const followerBadgeExtension = {
   filterExpression: 'isFollowerIndex:true',
 };
 
-extensionsService.addBadge(followerBadgeExtension);
+npSetup.plugins.indexManagement.extensionsService.addBadge(followerBadgeExtension);
```

Comment:
I think this is going to cause issues if index management is disabled.

Reply:
Good catch, I added a check.